axelrod/strategies/doubler.py

```diff
@@ -0,0 +1,34 @@
+from axelrod import Actions, Player
+
+C, D = Actions.C, Actions.D
+
+
+class Doubler(Player):
+    """
+    Cooperates except when the opponent defected and
+    opponent's cooperations are less than twice as often
```

Comment:
Could this be changed to:

```
and opponent's cooperation count is less than twice their defections.
```

@meatballs what do you think? Your judgement is way better than mine with these things.